tools/halide_image.h

```diff
@@ -223,9 +227,14 @@ class Image {
         contents->buf.min[2] = z;
         contents->buf.min[3] = w;
     }
+
+    void info(const char *tag = "Image") const;
+    void dump(const char *tag = "Image") const;
```

Comment:
They were originally standalone functions but were made members so that we could directly examine the private fields in Contents and Contents::buf.